### ISSUE TITLE
does not support all video formats

### ISSUE THREAD

**Issue by grunt2001c** [SoftProjector/softprojector#8]
Is it possible to play h264 mov and mp4 video formats?

**Comment by senechal-ua** [comment]
For now softprojector use codec packs, that install in your system (Windows)

**Comment by vladozar** [comment]
Yes, Currently softProjector uses system codec packages. Try installing http://www.codecguide.com/download_k-lite_codec_pack_basic.htm on Windows. I was able to play mp4 on Windows 7
